lib/AST/ASTImporter.cpp

```diff
@@ -4337,7 +4357,7 @@ Decl *ASTNodeImporter::VisitClassTemplateSpecializationDecl(
 
     D2->setTemplateSpecializationKind(D->getTemplateSpecializationKind());
 
-    // Set the context of this specialization/instantiation.
+    // Add the specialization to this context.
```

Comment:
Actually here, the original comment (the removed one) is the true one, since we don't add anything to the declcontext.